charts/argo-services/values.yaml

```diff
@@ -1,3 +1,4 @@
+awsAccountId: "1112223334"
```

Comment:
Does there need to be a value added for awsAccountID?

Reply:
There needs to dummy value  here for the tests. When w deploy this the actual AccountID will be passed from Terraform

Reply:
Maybe a comment here would be good idea.